cpp/data/system_updates/103.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -o errexit
+
+
+if [[ $TUEFIND_FLAVOUR != "krimdok" ]]; then
+    exit 0
+fi
+
+if [ -e /var/spool/cron/root ]; then # CentOS
+    root_crontab=/var/spool/cron/root
+else # Ubuntu
+    root_crontab=/var/spool/cron/crontabs/root
```

Comment:
Our development systems might not have a root crontab, therefore the file will not exist and then sed might fail. See 101.sh for a good example.

Reply:
Done